File: themes/crosh/colors/base16-supernova.js
// Base16 Supernova
// Scheme: @lbcnz (https://github.com/lbcnz/supernova-theme)

var color_scheme = {
        'base00': '#2d333b',
        'base01': '#373e47',
        'base02': '#545d68',
        'base03': '#909dab',
        'base04': '#adbac7',
        'base05': '#cdd9e5',
        'base06': '#e7f3f4',
        'base07': '#ffffff',
        'base08': '#d40059',
        'base09': '#ff5c00',
        'base0A': '#d9b600',
        'base0B': '#6bac00',
        'base0C': '#00a8e0',
        'base0D': '#237ef4',
        'base0E': '#00af9e',
        'base0F': '#ff66c2',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(205, 217, 229, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);

File: themes/pygments/pygments_base16/base16-supernova.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#2d333b'
    base01 = '#373e47'
    base02 = '#545d68'
    base03 = '#909dab'
    base04 = '#adbac7'
    base05 = '#cdd9e5'
    base06 = '#e7f3f4'
    base07 = '#ffffff'
    base08 = '#d40059'
    base09 = '#ff5c00'
    base0a = '#d9b600'
    base0b = '#6bac00'
    base0c = '#00a8e0'
    base0d = '#237ef4'
    base0e = '#00af9e'
    base0f = '#ff66c2'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('supernova', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords
